### DISCUSSION TITLE
Metrics from Prometheus plugin vs /status endpoint

### DISCUSSION THREAD

**Discussion by salomekbg** [Kong/kong#9075] [Help]
Hey folks,

I wanted to confirm if the `kong_nginx_http_current_connections` metrics with the various states in the [Promethus plugin](https://docs.konghq.com/hub/kong-inc/prometheus/#available-metrics) are the same as the `server connections` metrics from the `/status` endpoint mentioned [here](https://docs.konghq.com/gateway/latest/admin-api/#retrieve-node-status). 

Thank you!

**Comment by yzhan289** [comment]
Bump on this question.
